src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -125,6 +128,15 @@ ShouldUseSubqueryPushDown(Query *originalQuery, Query *rewrittenQuery)
 		return true;
 	}
 
+	/*
+	 * We handle outer joins as subqueries, since the join order planner
+	 * does not know how to handle them.
+	 */
+	if (FindNodeCheck((Node *) originalQuery->jointree, IsOuterJoinExpr))
```

Comment:
Just FYI for future reference and in case you've not realized since it is slightly implicit: We already used query pushdown planning for outer joins if the we have unsupported join clauses: https://github.com/citusdata/citus/pull/1918/files